### DISCUSSION TITLE
Division Formula [Apr 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1975] [Discord Threads]
> 🙋‍♂️ **Evasty's**  
#### formula column is:  (cost / days)  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/967880467443507310/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### Im using number types with your formula and it looks fine  
![image0](https://cdn.discordapp.com/attachments/967880468060057740/967991349993406535/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### please create an issue in github with details  
#  
> 🙋‍♂️ **Evasty's**  
#### ty, i did, not sure if it'll show as github flagged my acc as soon as it was made so i'll link it here:   
#### https://github.com/nocodb/nocodb/issues/1868#issue-1214836991  
#  
> 👨‍🚀 **wingkwong**  
#### i couldnt access this link  
#  
> 🙋‍♂️ **Evasty's**  
#### hope you don't mind me copying the issue here as i haven't been able to solve the account flag issue:   
####    
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **pg**   
#### ProjectOnRootDB: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.90.7**```   
####    
#### **Steps To Reproduce**   
####    
#### 1. Host on heroku with 1 click free deploy button (everything as is) and create a new project with internal db.   
#### 2. Add a table, add 2 number columns (cost, days), and any number of formula columns which use the number columns.   
#### 3. Add rows   
#### 4. The formula columns fail to show any result and there're no error reports.   
####    
####    
#### **Expected Behavior**   
####    
#### The formula columns to be filled or an error notification.   
####    
#### **Screenshots**   
#### ![image](https://user-images.githubusercontent.com/104383186/165142735-32979050-c1d6-4cbe-a4c4-8483824ad971.png)   
#### formulas are respectively:   
#### "MAX(cost, days)" , "cost" and "(cost / days)".   
####    
#### also noted that when downloading the sheet as csv all values are filled correctly.   
#### refreshing/reloading does nothing.   
#### (view of sample sheet: https://testlvl2nocodb.herokuapp.com/dashboard/#/nc/view/dc4cb9be-110e-4856-9d34-211bb78645f5 )  
#  
> 👨‍🚀 **wingkwong**  
#### lemme help create the issue for u  
#  
> 🙋‍♂️ **Evasty's**  
#### that's even better, thank you.   
#### if there's anything that might help please let me know.  
#  
> 👨‍🚀 **wingkwong**  
#### sure. created here - https://github.com/nocodb/nocodb/issues/1873  
#  
> 🙋‍♂️ **Evasty's**  
#### ty vm n.n!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/967880468060057740)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
